README.md

```diff
@@ -31,7 +31,8 @@ It allows you to easily create, maintain and execute automated web tests across
 
 Tests             | Status
 ----------------- | ----------
-All               | [![Build Status](https://travis-ci.org/DevExpress/testcafe.svg)](https://travis-ci.org/DevExpress/testcafe)
+All Travis tasks (server, client, functional: mobile, macOS, Edge)  | [![Travis CI Status](https://travis-ci.org/DevExpress/testcafe.svg)](https://travis-ci.org/DevExpress/testcafe) 
```

Comment:
Also add `[docs]` to PR title - we don't need full CI and we need to lint README